Docs/tuto_A_add_vehicle.md

```diff
@@ -111,7 +111,8 @@ __4.   Export the result__. Select all the meshes and the base of the skeleton a
 
 *   __11. Test the vehicle__. Launch CARLA, open a terminal in `PythonAPI/examples` and run the following command.  
 ```sh
-python3 manual_control.py --filter <model_name> # The name used in step 10.2
+python3 manual_control.py --filter vehicle.<make>.<model> # The names used in step 10.1 and 10.2
```

Comment:
I should maybe add that <make> and <model> need to be is lowercase?

Reply:
Hey, thanks for the corrections. Yes, you could put that by convention, make and model in CARLA uses lowercase characters